src/Tablet/Sensors/SensorsTab.module.css

```diff
@@ -1,3 +1,16 @@
-.column {
+.gridContainer {
   margin-top: 20px;
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-template-rows: auto auto;
+  grid-gap: 0px;
+  padding-right: 5%;
+  padding-left: 5%;
+}
+
+.gridItem {
+  background-color: #1b1b1b;
+  border: 4px outset white;
```

Comment:
It looks really ugly with this border 😆 

![image](https://user-images.githubusercontent.com/733074/61361295-89a3bb80-a880-11e9-8c31-2d789255afe8.png)